merging_health.gms

```diff
@@ -0,0 +1,111 @@
+$include data_0203
+$include parameters_0203
+
+* --- Defining the parameter to integrate CAPRI and Springmann et al. model:
+
+parameter cons_scn_data(*,*,*,*,*) "data on consumption scenarios";
+
+* --- Consumption data from Springmann et al. relates to human intake in CAPRI, so we first need to import reference data for human intake from CAPRI:
+
+execute_load "%results_in%\capmod\res_2_%BAS%%SIM%cap_after_2014_ref.gdx" p_DataOutTemp=dataout;
+
+* --- Consumption data from Springmann et al. is presented in g/d_w, kcal/d_w, g/d and kcal/d in the file cons_scn_data.gdx, but only g/d_w is used in the calculation of relative risks (RR_0203.gms).
+* --- Intake in g per capita per day (net of waste) by food product in the reference scenario (to be checked with Adrian: is all intake booked on 'N_CAL'?):
+
+     p_DataOutTemp(rall, "", "INCE", rows, "%SIMY%")
+     $ p_NCNC_CONT(rows, "N_CAL")
+     = p_DataOutTemp(rall, "", "N_CAL", rows,"%SIMY%") / p_NCNC_CONT(rows, "N_CAL") * 1000;
+
+* --- Food groups available in Springmann et al. are the following (26 fg): wheat, rice, maize, othr_grains, roots, sugar, legumes, soybeans, nuts_seeds, oil_veg, oil_palm, vegetables, fruits_trop, fruits_temp,
+* --- fruits_starch, beef, lamb, pork, poultry, eggs, milk, shellfish, fish_freshw, fish_pelag, fish_demrs, othrcrp. However, relative risks are calculated based only based on fruits (fruits_temp, fruits_trop, fruits_starch),
+* --- vegetables, nuts_seeds, legumes (legumes, soybeans) and red_meat (beef, lamb, pork)
+* --- To be checked with A.LEIP: names and level of aggregation of the food groups. In Springmann et al., there is also RR attributable to processed meat, which will have to be disactivated in the code saved in the /dat.
+
+      cons_scn_data(rall,"g/d_w","BMK",eatFoodGrp, "2010") = sum(eatFood2O(eatFoodGrp, rows), p_DataOutTemp(rall, "", "INCE", rows,"%SIMY%"));
+
+* --- Mapping CAPRI regions to Springmann et al (180 regions):
+
+      cons_scn_data(r,"g/d_w","BMK",eatFoodGrp, "2010") =  sum((map_reg(r, rall), cons_scn_data(rall,"g/d_w",eatFoodGrp, "2010"));
+
+* --- To be checked with A.LEIP: Before generating the final file, we need to create different diet scenarios for "FLX", "PSC", "VEG" and "VGN". The flexitarian dietary patterns contain no processed meat, low amounts of
+* --- red meat (including beef, lamb, and pork) and sugar, moderate amounts of poultry, dairy, and fish, and generous amounts of fruits, vegetables, legumes, and nuts. The other three dietary patterns replace meat
+* --- (pescatarian or vegetarian) or all animal source foods (vegan) with two-thirds either fish and seafood (pescatarian diets) or legumes (vegetarian and vegan diets) and a third fruits and vegetables. The reorganization of the
+* ---  different diet patterns should be done at the country level (to respect national preferences). How to do it in CAPRI?
+
+
+
+
+* --- Save results to files that will be linked to Springmann et al consumption:
```

Comment:
From here on i don't understand what the code is doing, as i can't see the files/data. 
Is RR_0203, PAF_0203, health_0203, report_0203 gams code? 
i remember we discussed some code from Marco earlier. is this original code then from Marco? no need to adapt for use in CAPRI?

Please add also to the repository, as otherwise it remains opaque what it is doing...